snpe/notebooks/non-marketplace_simulations.ipynb

```diff
@@ -0,0 +1,446 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c2b16772",
+   "metadata": {},
+   "source": [
+    "# Non-marketplace simulations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98992575",
+   "metadata": {},
+   "source": [
+    "### A. Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c95e1d8b",
+   "metadata": {},
+   "source": [
+    "In this notebook you will see how to run the four classes of non-marketplace simulations. The factor that differentiates these simulations is the number of Rho and Herding parameters considered in each of them. At this point it might be useful to formulate a brief reminder about the nature of the aforementioned parameters:\n",
+    "\n",
+    "- __Rho (ρ)__: Represents the cost of posting a rating measured in terms of the difference between the expected product experience and the actual product experience. The higher Rho is, the higher the difference between the expected and the actual experience will have to be for the user to leave a review about the product. \n",
+    "\n",
+    "\n",
+    "\n",
+    "- __Herding parameter__: A product-specific measurement of the probability of exhibiting herding behavior (i.e. be influenced by previous users' reviews at the time of posting a review). The higher that the herding parameter is, the more likely it is that the review of a user will be influenced by all previously posted reviews.\n",
+    "\n",
+    "\n",
+    "As mentioned, there are four different classes of non-marketplace simulations of increasing complexity defined by the number of Rho and herding parameters considered.\n",
+    "\n",
+    "1. __Single Rho Simulation__: It is represented by the SingleRhoSimulator class. This configuration considers the inclussion of a unique value for the rho paramter that is prevalent whether the actual product exprience was better or worse than expected. This implies that the cost of rating is the same irrespective of the sentiment behind such rating.  \n",
+    "\n",
+    "\n",
+    "2. __Double Rho Simulation__: It is represented by the DoubleRhoSimulator class, which is a child class of the single SingleRhoSImulator. This kind of simulation allows for the existence of two different rho parameters. One of these will apply for the cases when the the difference between the actual and the expected product experience is positive (denoted as ρ+) while the other applies in those cases where such difference is negative (denoted as ρ-). This kind of simulation allows you to consider different costs for leaving a review depending on the sentiment (positive/negative) behind it.\n",
```

Comment:
Only small typo: SingleRhoSimulator has an extra i capitalized in your text

Reply:
Typo corrected!